javascript/ecmascript-2015/number-extensions/README.md

```diff
@@ -4,6 +4,6 @@ description: Extending how Numbers work in ECMAScript
 
 insights:
   - math-extensions
-  - binary-and-octal-notation-in-es6
```

Comment:
cc @Loopiezlol any consequences to this? Anything else that might be affected?

Reply:
theoretically no

we first parse insights then workouts. so the insight change should get processed first (aka slug change) then this reference will properly be resolved